function-invokers/java-function-invoker/README.md

```diff
@@ -0,0 +1,13 @@
+Testing locally:
```

Comment:
should this also include instructions for a `Flux` function (requiring `--function.runner.isolated=true`)?

Reply:
Good point. But there is no README in master. Where are you seeing that?

Reply:
Got it (see latest commit).